easybuild/easyconfigs/s/STRUMPACK/STRUMPACK-6.1.0-foss-2020b.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'CMakeMake'
+
+name = 'STRUMPACK'
+version = '6.1.0'
+
+homepage = 'https://fastmath-scidac.llnl.gov/software/strumpack.html'
+description = """STRUMPACK - STRUctured Matrix PACKage - Fast linear solvers and preconditioner
+ for both dense and sparse systems using low-rank structured factorization with randomized sampling."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/pghysels/%(name)s/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['219ec7360594172464aafa6ecac1fd161097db6fb9ee35af5c1ca61531f4f5c4']
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('ParMETIS', '4.0.3'),
+    ('SCOTCH', '6.1.0'),
+]
+
+separate_build_dir = True
+
+configopts  = '-DCMAKE_BUILD_TYPE=Release '
+configopts += '-DCMAKE_CXX_COMPILER=mpicxx '
+configopts += '-DCMAKE_C_COMPILER=mpicc '
+configopts += '-DCMAKE_Fortran_COMPILER=mpifort '
```

Comment:
The options `separate_build_dir` and `CMAKE_BUILD_TYPE=Release` are default with CMakeMake. By also adding `usempi` to toolchainopts, the following can be removed:
```suggestion
```